WordPress/Classes/Services/Stories/StoryPoster.swift

```diff
@@ -0,0 +1,149 @@
+import Foundation
+import UIKit
+import WordPressKit
+import AutomatticTracks
+
+struct Story: Codable {
+    let mediaFiles: [MediaFile]
+}
+
+/// The contents of a Story block
+struct MediaFile: Codable {
+    let alt: String
+    let caption: String
+    let id: Double
+    let link: String
+    let mime: String
+    let type: String
+    let url: String
+
+    init(alt: String,
+         caption: String,
+         id: Double,
+         link: String,
+         mime: String,
+         type: String,
+         url: String) {
+        self.alt = alt
+        self.caption = caption
+        self.id = id
+        self.link = link
+        self.mime = mime
+        self.type = type
+        self.url = url
+    }
+
+    init(dictionary: [String: Any]) {
+        self.init(alt: dictionary["alt"] as! String,
+            caption: dictionary["caption"] as! String,
+            id: dictionary["id"] as! Double,
+            link: dictionary["link"] as! String,
+            mime: dictionary["mime"] as! String,
+            type: dictionary["type"] as! String,
+            url: dictionary["url"] as! String)
```

Comment:
⚠️ Force unwrap! Can a default value be returned instead?
Also, since the model is conforming to `Codable` and all the properties are required, should the Parser throw an error during the parsing?